### PR TITLE
Updated the OpenJcePlusTests to FIPS140_3_OpenJCEPlusFIPS and FIPS140_3_OpenJCEPlusFIPS.FIPS140-3

### PR DESCRIPTION
**Pull request by annaibm**
Signed-off-by: Anna Babu Palathingal <anna.bp@ibm.com>

related:https://github.com/adoptium/aqa-tests/issues/5605

### CODE REVIEW

File: functional/OpenJcePlusTests/build.xml [hunk 1]
```diff
@@ -123,7 +123,8 @@
 	<target name="build" >
 		<if>
 			<or>
-				<contains string="${TEST_FLAG}" substring="OpenJcePlusTests"/>
+				<contains string="${TEST_FLAG}" substring="FIPS140_3_OpenJCEPlusFIPS"/>
+				<contains string="${TEST_FLAG}" substring="FIPS140_3_OpenJCEPlusFIPS.FIPS140-3"/>
```
Comment (llxia) [inline functional/OpenJcePlusTests/build.xml:127]: `<contains string="${TEST_FLAG}" substring="FIPS140_3_OpenJCEPlusFIPS"/>` is sufficent. We do not need the second line

**Comment by llxia** [comment]
Please provide Grinder links.

**Comment by annaibm** [comment]
Grinder links:
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43177/
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43178/

**Comment by llxia** [comment]
The test is skipped in both Grinder.

**Comment by llxia** [comment]
For OpenJcePlusTests, you need to use Semeru build.

**Comment by annaibm** [comment]
Grinder links with Semeru builds
**s390x_linux**
TEST_FLAG: 
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43224/

TEST_FLAG:FIPS140_3_OpenJCEPlusFIPS
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43227/

**ppc64le_linux**
[https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43225/testReport/](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43225/rebuild/parameterized)

FIPS140_3_OpenJCEPlusFIPS
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43228/

**Comment by llxia** [comment]
- The PR title has a typo. The two flags are FIPS140_3_OpenJCEPlusFIPS and FIPS140_3_OpenJCEPlusFIPS.FIPS140-3
- Please run a Grinder with FIPS140_3_OpenJCEPlusFIPS.FIPS140-3 as well

**Comment by annaibm** [comment]
Grinder link:
FIPS140_3_OpenJCEPlusFIPS.FIPS140-3:-
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43235/

**Review by llxia — APPROVED** [review]
LGTM

**Review by LongyuZhang — APPROVED** [review]

**Pull request merged**